src/components/Navbar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <nav class="navbar">
+        <div class="logo">
+            <img :src="keyclashlogo" alt="KeyClash Logo">
+        </div>
+        <ul class="nav-links">
+            <li><a href="/">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#about-us">About us</a></li>
+        </ul>
+    </nav>
+</template>
+
+<script>
+import keyclashlogo from '@/public/keyclashlogo.png'
+
+export default {
+    name: 'Navbar',
+    data() {
+        return {
+            keyclashlogo
+        }
+    }
+}
+</script>
+
+<style scoped>
+@keyframes fadeIn {
```

Comment:
What does this notation mean? @keyframes, is this some sort of function or something different?